tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_suppliers.feature

```diff
@@ -684,3 +684,48 @@ Feature: Update product combination suppliers in Back Office (BO)
     Then product product7 should not exist anymore
     And supplier "supplier4" should have 0 products associated
     And supplier "supplier5" should have 0 products associated
+
+  Scenario: Product suppliers can be updated when some combinations doesn't exist in default shop
```

Comment:
This should be into a feature dedicated to multishop, I prefer we keep a clean separation of multishop features, so that the background that creates/enables shops is unique and always at the top of the file And to make sure we clean the data correctly The `@restore-shops-after-feature` is not in this file, so it will likely impact the other tests sooner or later

So this should be moved into the shop features, and since we don't have features for suppliers in multishop yet you'll likely have to add one in `tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/update_multishop_combination_suppliers.feature`